src/module.c

```diff
@@ -8091,14 +8112,27 @@ int moduleTryServeClientBlockedOnKey(client *c, robj *key) {
  *     free_privdata:    called in order to free the private data that is passed
  *                       by ValkeyModule_UnblockClient() call.
  *
- * Note: ValkeyModule_UnblockClient should be called for every blocked client,
- *       even if client was killed, timed-out or disconnected. Failing to do so
- *       will result in memory leaks.
+ * Notes:
+ * 1. ValkeyModule_UnblockClient should be called for every blocked client,
+ * even if client was killed, timed-out or disconnected. Failing to do so
+ * will result in memory leaks.
+ * 2. Blocking the client on keyspace event notification is supported from
+ * version 8.1
```

Comment:
If we want to consider this a bugfix and release it in a patch version, then we should formulate it differently to comply with [semver](https://semver.org/). Something like this?

```suggestion
 * 2. Attempting to block the client on keyspace event notification in versions
 *    prior to 8.1.1 leads to a crash.
```

Reply:
Done

Reply:
Apparently this change is not pushed yet.

Reply:
oops, fixed now.